packages/test/src/test-ai-sdk.ts

```diff
@@ -0,0 +1,322 @@
+/**
+ * Test AI SDK integration with Temporal workflows
+ */
+import { LanguageModelV2Content, LanguageModelV2FinishReason } from '@ai-sdk/provider';
+import { openai } from '@ai-sdk/openai';
+import { v4 as uuid4 } from 'uuid';
+import * as opentelemetry from '@opentelemetry/sdk-node';
+import { SEMRESATTRS_SERVICE_NAME } from '@opentelemetry/semantic-conventions';
+import { ExportResultCode } from '@opentelemetry/core';
+import { StdioClientTransport } from '@modelcontextprotocol/sdk/client/stdio.js';
+import { experimental_createMCPClient as createMCPClient } from '@ai-sdk/mcp';
+import { AiSDKPlugin, ModelResponse, TestProvider } from '@temporalio/ai-sdk';
+import { temporal } from '@temporalio/proto';
+import { WorkflowClient } from '@temporalio/client';
+import {
+  makeWorkflowExporter,
+  OpenTelemetryActivityInboundInterceptor,
+  OpenTelemetryActivityOutboundInterceptor,
+  OpenTelemetrySinks,
+  OpenTelemetryWorkflowClientCallsInterceptor,
+  OpenTelemetryWorkflowClientInterceptor,
+} from '@temporalio/interceptors-opentelemetry';
+import { InjectedSinks, Runtime } from '@temporalio/worker';
+import {
+  generateObjectWorkflow,
+  helloWorldAgent,
+  mcpWorkflow,
+  middlewareWorkflow,
+  telemetryWorkflow,
+  toolsWorkflow,
+} from './workflows/ai-sdk';
+import { helpers, makeTestFunction } from './helpers-integration';
+import { getWeather } from './activities/ai-sdk';
+import { Worker } from './helpers';
+import EventType = temporal.api.enums.v1.EventType;
+
+const remoteTests = !!process.env.AI_SDK_REMOTE_TESTS;
```

Comment:
Do you know that `!!"false"` is true?

Please use the following idiom instead (or look at `isSet` in `worker-options.ts`).
That will ensure that this variable follows the same thruthness conventions as our other env variables.

```suggestion
const remoteTests = ['1', 't', 'true'].includes((process.env.AI_SDK_REMOTE_TESTS ?? 'false').toLowerCase()),
```